datasets/kwaliteitsmonitor/kwaliteitsmonitorWonenLigplaats/v1.0.0.json

```diff
@@ -0,0 +1,169 @@
+{
+  "id": "kwaliteitsmonitorWonenLigplaats",
```

Comment:
En kan dit `ligplaats` worden? Ik vermoed dat Postgres hier ook moeilijk gaat doen omdat de identifiers te lang worden:

    kwaliteitsmonitor__kwaliteitsmonitor_wonen_ligplaats__bag_typeadresseerbaarobject

is 81 tekens en de limiet is 63. Zonder `kwaliteitsmonitorWonen` past het net, maar korter is beter.